vector/src/main/java/im/vector/riotx/features/home/room/detail/composer/TextComposerView.kt

```diff
@@ -86,8 +87,8 @@ class TextComposerView @JvmOverloads constructor(context: Context, attrs: Attrib
         }
 
         sendButton.setOnClickListener {
-            val textMessage = text?.toString() ?: ""
-            callback?.onSendMessage(textMessage)
+            val textMessage = text?.toSpannable()
+            callback?.onSendMessage(textMessage ?: "")
```

Comment:
I prefer to add `?: ""` at the end of the previous line, for code clarity

Reply:
Done

Reply:
Ok, I was not clear, I'll do it myself in a commit :)